net-misc/ptpd/ptpd-2.3.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit autotools eutils flag-o-matic systemd
+
+DESCRIPTION="Precision Time Protocol daemon"
+HOMEPAGE="http://ptpd.sf.net"
+
+if [[ ${PV} == "9999" ]]; then
```

Comment:
While I don't mind strongly having live code there, it feels a bit weird to not have -9999 ebuild that could use it :).